.github/workflows/ci.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       run: sudo apt-get install gcc-multilib
 
     - name: Pin deps that break MSRV
-      if: matrix.rust_version == '1.70.0'
+      if: matrix.rust_version == '1.71.0'
       run: |
         cargo update -p ahash --precise 0.8.7
         cargo update -p bumpalo --precise 3.14.0
```

Comment:
Looks like neither of these should be needed now? And the first is causing CI errors.